lib/api/request/kuzzleRequest.ts

```diff
@@ -604,6 +604,34 @@ export class KuzzleRequest {
     return this._getObject(this.input.args, name, name, def);
   }
 
+  /**
+   * Gets a parameter from a request arguments and returns it to the appropriate format.
+   *
+   * @param name parameter name.
+   * @param options Additional options
+   *    - `format`: date format timestamp, date or ISO8061 (default: 'date')
+   * @throws {api.assert.missing_argument} If parameter not found and no default
+   *                                       value provided
+   */
+  getDate (name: string,
+    options: {
+            format?: 'timestamp' | 'date' | 'ISO8061';
+          } = { format: 'date' }
+  ): number | Date | string {
```

Comment:
Nitpicking: coding standard: when the line is too long, each argument should be on it's own line
```suggestion
function getDate (
  name: string,
  options: { format?: 'timestamp' | 'date' | 'ISO8061' } = { format: 'date' },
): number | Date | string { }
```